spec/mailers/distribution_mailer_spec.rb

```diff
@@ -103,4 +103,36 @@
       end
     end
   end
+  describe "#requestee_email assignment" do
+    let(:requestee_email) { "requestee@example.com" }
+    let(:partner) { create(:partner) }
+    let(:organization) { create(:organization) }
+    let!(:user) { create(:partner_user, email: requestee_email, partner: partner) }
+    let!(:request) { create(:request, partner: partner, organization: organization, partner_user_id: user.id) }
+  
+    context "when distribution has a request with a associated user" do
+      let!(:distribution) { create(:distribution, partner: partner, organization: organization, request: request) }
+      it "assigns the requestee's email from the request's user_email" do
+        requestee_email_result = distribution.request ? distribution.request.user_email : partner.email
+        expect(requestee_email_result).to eq(requestee_email)
+      end
+    end
+
+    context "when distribution has a request without an associated user" do
+      let!(:request) { create(:request, partner: partner, organization: organization, partner_user_id: nil) }
+      let!(:distribution) { create(:distribution, partner: partner, organization: organization, request: request) }
+      it "assigns the requestee's email from the request's user_email" do
+        requestee_email_result = distribution.request ? distribution.request.user_email : partner.email
+        expect(requestee_email_result).to eq(partner.email)
+      end
+    end
+  
+    context "when distribution doesn't have a request" do
+      let!(:distribution_without_request) { create(:distribution, partner: partner, organization: organization, request: nil) }
+      it "assigns the requestee's email from the partner's email" do
+        requestee_email_result = distribution_without_request.request ? distribution_without_request.request.user_email : partner.email
+        expect(requestee_email_result).to eq(partner.email)
+      end
+    end
+  end
```

Comment:
Hey - no none of these actually test that the requestee is the one getting the e-mail :)

Reply:
oof critical mistake

Reply:
ill take care of the linting issues also lol

Reply:
Something like that, yep!